brownie-config.yml

```diff
@@ -1,7 +1,7 @@
 # use Ganache's forked mainnet mode as the default network
 # NOTE: You don't *have* to do this, but it is often helpful for testing
 networks:
-  default: binance-fork
+  default: bsc-main-fork
```

Comment:
This was changed but on ci the name of the network wasnt changed,changing the ci bsc fork network should fix the test.